### ISSUE TITLE
Warning: Accessing non-existent property '' of module exports inside circular dependency

### ISSUE THREAD

**Issue by fishcharlie** [twilio/twilio-cli#178]
### Issue Summary

When upgrading to Node.js v14.0.0 and running **any** Twilio command it prints a bunch of warnings.

This should be fixed to not print any warnings on a valid Twilio command.

Related Node.js Issue: https://github.com/nodejs/node/issues/32987

### Steps to Reproduce
1. Upgrade to Node.js > 14.0.0
2. Run `twilio`

### Code Snippet

NA

### Exception/Log
```
charliefish@Charlies-MacBook-Pro ~ % twilio
(node:51510) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:51510) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'set' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:51510) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
unleash the power of Twilio from your command prompt

VERSION
  twilio-cli/2.0.0 darwin-x64 node-v14.0.0

USAGE
  $ twilio [COMMAND]

COMMANDS
  api            advanced access to all of the Twilio APIs
  autocomplete   display autocomplete installation instructions
  debugger       Show a list of log events generated for the account
  email          sends emails to single or multiple recipients using Twilio SendGrid
  feedback       provide feedback to the CLI team
  help           display help for twilio
  login          create a new profile to store Twilio Account credentials and configuration
  phone-numbers  manage Twilio phone numbers
  plugins        list installed plugins
  profiles       manage credentials for Twilio profiles
```

### Technical details:
* twilio-cli version: twilio-cli/2.0.0
* node version: node-v14.0.0
* Command output with debug logging enabled (adding `-l debug` to the end of the command):
```
charliefish@Charlies-MacBook-Pro ~ % twilio -l debug
(node:51685) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:51685) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'set' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:51685) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
 ›   Error: command -l not found
```

**Comment by childish-sambino** [comment]
Looks like this was fixed by a dependency update in the 2.0.1 release. If you're able to upgrade to 2.0.1, let me know if you're still seeing it.

**Comment by fishcharlie** [comment]
Closing this issue as Twilio CLI v2.0.1 fixes this. Thanks @childish-sambino!!
